semgrep/tests/e2e/snapshots/test_baseline/test_some_intersection/baseline_error.txt

```diff
@@ -1,2 +1,6 @@
 Running baseline scan with base set to: baseline-commit
+
+skipped: all .gitignored files, 1 files larger than 1.0 MB
```

Comment:
@brendongo any idea why there's a 1+ MB large file in this test?

Reply:
hmmm there def shouldnt be

Reply:
Ah I think this is some issue with TargetManager caching things. Bug with --baseline-commit but since that is experimental non blocking for this PR. Will follow up with issue + fix

Reply:
Tracked https://linear.app/r2c/issue/PA-844/semgrep-baseline-commit-file-size-filtering-bug